integration/helpers.go

```diff
@@ -171,6 +173,8 @@ type InstanceConfig struct {
 
 	// log specifies the logger
 	log utils.Logger
+
+	DisableKubeImpersonationPermissionsCheck bool
```

Comment:
This looks very out of place in InstanceConfig and in TeleInstance to be honest. Can tests just always disable it when creating a kube forwarder (by passing a dummy implementation as I suggested below) or are there tests that actually need it?

Reply:
Done